keepassxc-browser/content/keepassxc-browser.js

```diff
@@ -1013,7 +1054,8 @@ kpxc.siteIgnored = function(condition) {
     return false;
 };
 
-kpxc.initCredentialFields = async function(forceCall, inputs) {
+// Initializes all input fields from the whole page
+kpxc.initCredentialFields = async function(forceCall) {
```

Comment:
This function signature changed but don't see where any calls to it changed or where inputs was dropped from use.

Reply:
See line 827 inside `kpxcObserverHelper.handleObserverAdd()`. The whole `inputs` parameter was added to `kpxc.initCredentialFields()` because of the input fields added by the observer.